examples/affinity/README.adoc

```diff
@@ -0,0 +1,124 @@
+
+[source]
+----
+oc get nodes --show-labels
+----
+
+The output the command would look like the following output. As you can see, the only nodes that are labeled with `disktype=ssd` are `worker0` and `worker1` nodes.
+
+[source]
+----
+oc get nodes --show-labels
+NAME                             STATUS   ROLES    AGE   VERSION   LABELS
+master0.shames.os.fyre.ibm.com   Ready    master   26d   v1.16.2   beta.kubernetes.io/arch=amd64,...
+master1.shames.os.fyre.ibm.com   Ready    master   26d   v1.16.2   beta.kubernetes.io/arch=amd64,...
+master2.shames.os.fyre.ibm.com   Ready    master   26d   v1.16.2   beta.kubernetes.io/arch=amd64,...
+worker0.shames.os.fyre.ibm.com   Ready    worker   26d   v1.16.2   disktype=ssd,beta.kubernetes.io/arch=amd64...
+worker1.shames.os.fyre.ibm.com   Ready    worker   26d   v1.16.2   disktype=ssd,beta.kubernetes.io/arch=amd64...
+worker2.shames.os.fyre.ibm.com   Ready    worker   26d   v1.16.2   beta.kubernetes.io/arch=amd64,...
+----
+
+In this cluster, we want the `coffeeshop-frontend` and `coffeeshop-backend` applications to be co-located on the same nodes with SSD storage types.
+
+Here is a YAML snippet of the `RuntimeComponent` custom resource (CR) for the `coffeeshop-backend` application with two replicas. The custom resource has `.spec.affinity.podAntiAffinity` configured to ensure the scheduler does not co-locate pods for the application on a single node. As described in the link:++https://github.com/application-stacks/runtime-component-operator/blob/master/doc/user-guide.adoc#labels++[user guide], the application pods are labelled with `app.kubernetes.io/instance: metadata.name`.
+
+The custom resource also has `spec.affinity.nodeAffinityLabels` to ensure the application pods will run on nodes with the `disktype: ssd` label.
+
+[source,yaml]
+----
+apiVersion: app.stacks/v1beta1
+kind: RuntimeComponent
+metadata:
+  name: coffeeshop-backend
+spec:
+  applicationImage: 'k8s.gcr.io/pause:2.0'
+  replicas: 2
+  affinity:
+    nodeAffinityLabels:
+      disktype: ssd
+    podAntiAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        - labelSelector:
+            matchExpressions:
+              - key: app.kubernetes.io/instance
+                operator: In
+                values:
+                  - coffeeshop-backend
+          topologyKey: kubernetes.io/hostname
+----
+
+The YAML snippet below shows the `RuntimeComponent` CR for the `coffeeshop-frontend` application. The snippet has the `.spec.affinity.podAffinity` configured which informs the scheduler that all its replicas are to be co-located with pods that have selector label `app.kubernetes.io/instance: coffeeshop-backend`. The CR also includes `.spec.affinity.podAntiAffinity` to ensure that each `coffeeshop-frontend` replica does not co-locate on a single node. In addition, the `spec.affinity.nodeAffinityLabels` parameter is specified to ensure the application pods will run on nodes with the `disktype: ssd` label.
```

Comment:
(Note: Avoid below, above, left, right to give direction on a page. The words result in an accessibility violation because screen readers do not have a sense of direction.)
Change "The YAML snippet below shows..." to "The following YAML snippet shows...".

Change "...`.spec.affinity.podAffinity` configured..." to '...`.spec.affinity.podAffinity` field configured...".

Change "...have selector label `app.kubernetes.io/instance: coffeeshop-backend`." to "have the `app.kubernetes.io/instance: coffeeshop-backend` selector label.".

Chanage "...includes `.spec.affinity.podAntiAffinity` to..." to "... includes the `.spec.affinity.podAntiAffinity` label to...".

Change "....specified to ensure the application pods will run on nodes with the `disktype: ssd` label." to "...specified to ensure that the application pods run on nodes with the `disktype: ssd` label.".  <= We avoid future tense.